tools/startup/setup.sh

```diff
@@ -0,0 +1,115 @@
+  sudo apt-get install -fdy code
+  sudo apt-get -dy upgrade
+  # TODO: Figure out how to cache the "apt-get update" to the downloads folder.
+  cp /var/cache/apt/archives/*.deb downloads/apt
+fi
+sudo cp downloads/apt/*.deb /var/cache/apt/archives/
+sudo apt-get install -y git default-jdk pkg-config zip g++ zlib1g-dev unzip python python3 curl apt-transport-https
+sudo dpkg -i downloads/gconf2-common_3.2.6-4ubuntu1_all.deb downloads/libgconf-2-4_3.2.6-4ubuntu1_amd64.deb
+sudo install -o root -g root -m 644 downloads/microsoft.gpg /etc/apt/trusted.gpg.d/
+sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/vscode stable main" > /etc/apt/sources.list.d/vscode.list'
+sudo apt-get -y update
+sudo apt-get install -y code
+sudo apt-get -y upgrade
+
+# Install Chrome
+if [ ! -f downloads/chrome.deb ]; then
+  wget -O downloads/chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+fi
+sudo dpkg -i downloads/chrome.deb
+
+# Install Bazel
+if [ ! -f downloads/bazel.0.21.0.sh ]; then
+  wget -O downloads/bazel.0.21.0.sh https://github.com/bazelbuild/bazel/releases/download/0.21.0/bazel-0.21.0-installer-linux-x86_64.sh
+  chmod +x downloads/bazel.0.21.0.sh
+fi
+sudo bash downloads/bazel.0.21.0.sh --user
+
+# Install IntelliJ
+if [ ! -f downloads/idea.2018.3.3.tar.gz ]; then
+  wget -O downloads/idea.2018.3.3.tar.gz https://download.jetbrains.com/idea/ideaIC-2018.3.3.tar.gz
+  rm -rf opt/idea
+  mkdir -p opt/idea
+  tar xfz downloads/idea.2018.3.3.tar.gz -C opt/idea --strip 1
+fi
+
+# Set up desktop defaults
+gsettings set org.gnome.shell favorite-apps "['google-chrome.desktop', 'code.desktop', 'org.gnome.Nautilus.desktop', 'org.gnome.gedit.desktop', 'org.gnome.Terminal.desktop']"
+gsettings set org.gnome.desktop.interface show-battery-percentage true
+gsettings set org.gnome.desktop.interface enable-animations false
+gsettings set org.gnome.gedit.preferences.editor auto-save true
+gsettings set org.gnome.gedit.preferences.editor display-line-numbers true
+gsettings set org.gnome.gedit.preferences.editor insert-spaces true
+gsettings set org.gnome.gedit.preferences.editor tabs-size 2
+gsettings set org.gnome.nautilus.preferences show-hidden-files true
+gsettings set org.gtk.Settings.FileChooser show-hidden true
+rm -rf ~/Public
+rm ~/examples.desktop
+rm ~/Desktop/ubiquity.desktop 
+sudo touch ~/Templates/"Empty file.txt"
+
+# Set up .bashrc:
+echo "source /media/$USER/data/dotfiles/.terminal" >> ~/.bashrc
+# FIXME: Why does it not add to PATH:
+source ~/.bashrc
+
+# Set up StartupOS:
+git clone https://github.com/google/startup-os.git downloads/startup-os
+cd downloads/startup-os
+bazel run //tools/reviewer/aa:aa_tool -- init --base_path /media/$USER/data/base
+cd /media/$USER/data/base/head/startup-os
+./test.sh
+# FIXME: Why does it not add to PATH:
+source ~/.bashrc
+aa
+echo "Please sign-in at https://web-login-startupos.firebaseapp.com and then check that ~/.aa_token exists"
```

Comment:
@oferb from this line I understand your goal is to start `localserver`. What if we did it explicitly (put smth like [these lines](https://github.com/google/startup-os/blob/master/tools/reviewer/aa/aa_tool.sh#L126-L127))?

Reply:
We could, but why?
To avoid the "Unknown command"?

Reply:
@oferb yes. also, explicit is better than implicit

Reply:
Cool, added a TODO.